extensions/amp-ad-network-fake-impl/0.1/amp-ad-network-fake-impl.js

```diff
@@ -60,12 +63,18 @@ export class AmpAdNetworkFakeImpl extends AmpA4A {
 
   /** @override */
   getAdUrl() {
-    return this.element.getAttribute('src');
+    return (
```

Comment:
Is `srcdoc` a proper URL?

Reply:
no, you are right it's actually a string of document html, but I need the value in `sendXhrRequest`. It's a bit of a hack but I would prefer to not change the a4a impl. Open to suggestions if you think there is a better way.

Reply:
I feel overloading the variable `adUrl` could be misleading down the road.

How about we convert srcdoc to a data URL here? Maybe we do not need sendXhrRequest change or  `this.creativeInlined_` going this way.

Reply:
Good idea! Done.